pkg/processor/cascadingfilterprocessor/sampling/drop_trace_factory.go

```diff
@@ -23,15 +24,38 @@ import (
 	"github.com/SumoLogic/sumologic-otel-collector/pkg/processor/cascadingfilterprocessor/config"
 )
 
+const (
+	StatusCodeError = "Error"
+	StatusCodeOk    = "Ok"
+	StatusCodeUnset = "Unset"
+)
+
 type dropTraceEvaluator struct {
 	numericAttr *numericAttributeFilter
 	stringAttr  *stringAttributeFilter
 	attrs       []attributeFilter
 	operationRe *regexp.Regexp
+	statusCode  *string
 
 	logger *zap.Logger
 }
 
+func validateStatusCode(statusCode *string) error {
+	if statusCode == nil {
+		return nil
+	}
+
+	validStatusCodes := []string{StatusCodeError, StatusCodeOk, StatusCodeUnset}
```

Comment:
```suggestion
	validStatusCodes := []string{
	  ptrace.StatusCodeError.String(),
	  ptrace.StatusCodeOk.String(),
	  ptrace.StatusCodeUnset.String(),
	}
```

I would also generate error out of this array

Reply:
done!

Reply:
Can we use it in error string as well?

Reply:
yep fixed!